internal/pkg/cli/pipeline_delete.go

```diff
@@ -182,28 +182,33 @@ func (o *deletePipelineOpts) getSecret() error {
 	// Look for default secret name for GHv1 access token based on default pipeline name.
 	o.ghAccessTokenSecretName = o.pipelineSecretName()
 	output, err := o.secretsmanager.DescribeSecret(o.ghAccessTokenSecretName)
-	if err == nil {
-		// Check for Copilot-assigned tag for added assurance. If tags not found, fall through.
-		for _, tag := range output.Tags {
-			if aws.StringValue(tag.Key) == "copilot-application" && aws.StringValue(tag.Value) == output.CreatedDate.UTC().Format(time.UnixDate) {
-				log.Infof("Found secret '%s'.\n", o.ghAccessTokenSecretName)
-				return nil
-			}
+	if err != nil {
+		var notFoundErr *secretsmanager.ErrSecretNotFound
+		if errors.As(err, &notFoundErr) {
+			o.ghAccessTokenSecretName = ""
+			return nil
 		}
-	}
-	var notFoundErr *secretsmanager.ErrSecretNotFound
-	if err != nil && !errors.As(err, &notFoundErr) {
 		return fmt.Errorf("describe secret %s: %w", o.ghAccessTokenSecretName, err)
 	}
-	// To get here, either the secret was found but tags didn't match, or Secrets Manager returned a ResourceNotFoundException.
-	log.Infof("Found no Copilot-generated Secrets Manager secrets to delete for pipeline '%s'.\n", o.name)
+
+	for _, tag := range output.Tags {
+		if aws.StringValue(tag.Key) == deploy.AppTagKey && aws.StringValue(tag.Value) == output.CreatedDate.UTC().Format(time.UnixDate) {
+			log.Infof("Found the Copilot-generated secret '%s' used to connect to your GitHub source repo.\n", o.ghAccessTokenSecretName)
```

Comment:
I feel like since we are going to prompt them to confirm the removal, we can even get rid of this log